libdocker/libdocker.go

```diff
@@ -185,9 +186,8 @@ func AllInspectedContainers(endpoint string) ([]*CompleteDockerContainer, error)
 		go func(container *CompleteDockerContainer) {
 			defer wg.Done()
 
-			fullDescContainer, err := client.InspectContainer(shortDescContainer.ID)
+			fullDescContainer, err := client.InspectContainer(container.ID)
```

Comment:
I am not sure if `shortDescContainer.ID == fullDescContainer.ID`. If they are the same then this is OK.